test_traveling_sales.py

```diff
@@ -17,49 +17,35 @@ def test_anonymize_nrp(self):
         self.assertEqual('My NRP is Unknown',
                          anonymize_text('My NRP is Unknown', ['NRP']))
         
-    def test_au_medicare(self):
```

Comment:
original code. not sure if this is necessarily something you should be concerned with.